### DISCUSSION TITLE
Automatic Integration of ToC into Document

### DISCUSSION THREAD

**Discussion by DreiBaer** [l1xnan/obsidian-better-export-pdf#298] [Q&A]
Hey,
I'm looking for a plugin which will create a table of contents into my document when exporting it into PDF.
As I got an information from a community I thought this plugin is a better version of the default export. But I cannot find the solution to insert a TOC into my document directly when export the note into a PDF.

What do I have to do in my note or what is neccessary in the settings of Better Export PDF?

Best Regards
DreiBaer

**Comment by l1xnan** [comment]
You can try using other plugins to generate a table of contents. For example: [obsidian-automatic-table-of-contents](https://github.com/johansatge/obsidian-automatic-table-of-contents)
